### PR TITLE
summary of changes in the release

### PR DESCRIPTION
**Pull request by katerin71**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -107,11 +107,11 @@ Versions in Linux distributions:
...
 * [1.4.1](Release-Notes-1.4.1) (2017-04-26)
 * [1.3.5](Release-Notes-1.3.5) - only for EL6
-* [1.3.4](Release-Notes-1.3.4)
-* [1.3.3](Release-Notes-1.3.3) (2017-01-01) - bugfixing; a new config option for the builder hostname; upgraded temporary directories, chroot contains `best=1`
+* [1.3.4](Release-Notes-1.3.4) - new extension for .log `available_pkgs.log` and `installed_pkgs.log`, new configuration files, bugfixing
+* [1.3.3](Release-Notes-1.3.3)
 * [1.3.2](Release-Notes-1.3.2)
 * [1.2.21](Release-Notes-1.2.21)
-* [1.2.20](Release-Notes-1.2.20) (2016-08-17) - just a bugfix release, which uses correct gpg keys for epel in epel* configs.
+* [1.2.20](Release-Notes-1.2.20)
```
Comment (praiskup) [inline docs/index.md:114]: Please don't remove this.

File: docs/index.md [hunk 2]
```diff
@@ -107,11 +107,11 @@ Versions in Linux distributions:
 * [1.4.2](Release-Notes-1.4.2) (2017-06-20)
 * [1.4.1](Release-Notes-1.4.1) (2017-04-26)
 * [1.3.5](Release-Notes-1.3.5) - only for EL6
-* [1.3.4](Release-Notes-1.3.4)
-* [1.3.3](Release-Notes-1.3.3) (2017-01-01) - bugfixing; a new config option for the builder hostname; upgraded temporary directories, chroot contains `best=1`
+* [1.3.4](Release-Notes-1.3.4) - new extension for .log `available_pkgs.log` and `installed_pkgs.log`, new configuration files, bugfixing
```
Comment (praiskup) [inline docs/index.md:110]: Can you add (2017-02-27) becaue `AuthorDate: Mon Feb 27 17:26:04 2017 +0100`

Comment (katerin71) [reply docs/index.md]: I added the date.

File: docs/index.md [hunk 3]
```diff
@@ -107,11 +107,11 @@ Versions in Linux distributions:
 * [1.4.2](Release-Notes-1.4.2) (2017-06-20)
 * [1.4.1](Release-Notes-1.4.1) (2017-04-26)
 * [1.3.5](Release-Notes-1.3.5) - only for EL6
-* [1.3.4](Release-Notes-1.3.4)
-* [1.3.3](Release-Notes-1.3.3) (2017-01-01) - bugfixing; a new config option for the builder hostname; upgraded temporary directories, chroot contains `best=1`
+* [1.3.4](Release-Notes-1.3.4) -(2017-02-27) new extension for .log `available_pkgs.log` and `installed_pkgs.log`, new configuration files, bugfixing
```
Comment (praiskup) [inline docs/index.md:110]: please use "use `.log` extension for `available_pkgs` and `installed_pkgs` log files, support for custom nspawn args, new hw_plugin, `best=1` used for Rawhide, added Fedora 26 configs

**Pull request merged**